broker/lib/fabrik/Fabrik.js

```diff
@@ -1,73 +1,47 @@
 'use strict';
 
-const assert = require('assert');
-const Promise = require('bluebird');
-const config = require('../../../common/config');
-const catalog = require('../../../common/models').catalog;
 const Agent = require('../../../data-access-layer/service-agent');
-const VirtualHostAgent = require('./VirtualHostAgent');
-const DirectorManager = require('./DirectorManager');
-const VirtualHostManager = require('./VirtualHostManager');
 const DBManager = require('./DBManager');
 const OobBackupManager = require('./OobBackupManager');
-const BasePlatformManager = require('../../../platform-managers/BasePlatformManager');
+const catalog = require('../../../common/models/catalog');
+const DirectorService = require('../../../operators/bosh-operator/DirectorService');
+const DockerService = require('../../../operators/docker-operator/DockerService');
+const VirtualHostService = require('../../../operators/virtualhost-operator/VirtualHostService');
 const CONST = require('../../../common/constants');
-const DockerManager = config.enable_swarm_manager ? require('./DockerManager') : undefined;
+const config = require('../../../common/config');
 
 class Fabrik {
-  static createManager(plan) {
-    return Promise
-      .try(() => {
-        switch (plan.manager.name) {
-        case CONST.INSTANCE_TYPE.DIRECTOR:
-          return DirectorManager;
-        case CONST.INSTANCE_TYPE.DOCKER:
-          if (config.enable_swarm_manager) {
-            return DockerManager;
-          } else {
-            assert.fail(plan.manager.name, [CONST.INSTANCE_TYPE.DIRECTOR, CONST.INSTANCE_TYPE.VIRTUAL_HOST], undefined, 'in');
-          }
-          break;
-        case CONST.INSTANCE_TYPE.VIRTUAL_HOST:
-          return VirtualHostManager;
-        default:
-          assert.fail(plan.manager.name, [CONST.INSTANCE_TYPE.DIRECTOR, CONST.INSTANCE_TYPE.DOCKER, CONST.INSTANCE_TYPE.VIRTUAL_HOST], undefined, 'in');
-        }
-      })
-      .then(managerConstructor => managerConstructor.load(plan));
-  }
-
-  static createInstance(instance_id, service_id, plan_id, context) {
```

Comment:
/manage route still had assigninstance as a middle layer. I think we can get rid of that.